syntax = "proto3";
package app.protos;

option go_package = "protos/parser";

enum Markets {
  OZON = 0;
}

enum Filters {
  RANGE = 0;
  SELECTION = 1;
  SELECTIONRADIO = 2;
  BOOL = 3;
}

message RangeFilter {
  double min = 1;
  double max = 2;
}

message SelectionFilterItem {
  string text = 1;
  string value = 2;
}

message SelectionFilter {
  bool isRadio = 1;
  repeated SelectionFilterItem items = 2;
}

message BoolFilter {
  string value = 1;
}

message StringList {
  repeated string values = 1;
}